index.html

```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    <title>Pick a name, any name!</title>
+  </head>
+  <body>
+    <h1>Welcome!</h1>
```

Comment:
I'd like to see another layer of separation in your html, using perhaps Header, Main and Footer tags?

That will give you more flexibility to manipulate those block elements using CSS and JS.